.github/workflows/ci-model-regression.yml

```diff
@@ -460,6 +460,22 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
+      - name: Download an artifact with the results for the schedule version
+        run: |
+          # Schedule workflows ID - 1498790
```

Comment:
is there a way we can fetch the ID for the _latest_ or _current_ version of `ci-model-regression-on-schedule.yml`? this is not ideal as it needs to be manually updated if the other file changes.

if it has to happen via ID, we need to add comments here that the ID has to be updated if `ci-model-regression-on-schedule.yml` changes, and also in `ci-model-regression-on-schedule.yml` to update the ID in this file

Reply:
also we need to think about the case where another PR changes `ci-model-regression-on-schedule.yml`, so simply getting the latest ID wouldn't be right. can we fetch the latest workflow ID of that workflow for the current branch?

Reply:
The IDs for workflows are pretty static, they don't change even there are several branches that have different versions of the same workflow, additionally, schedule workflows can exist only in the master branch.

I've changed a bit this part and now, we fetch ID via GH API for the scheduled workflow.